setup.py

```diff
@@ -54,7 +50,7 @@
 
 setup(
     name='google-gax',
-    version=version,
+    version='0.15.5',
```

Comment:
Are you planning to make a new patch release just for this? It seems like an implementation detail to me.